src/aatoolbox/utils/raster.py

```diff
@@ -445,3 +462,107 @@ def compute_raster_statistics(
 
         df_zonal_stats = pd.concat(df_list).reset_index(drop=True)
         return df_zonal_stats
+
+
+@xr.register_dataset_accessor("aat")
+class AatRasterDataset(AatRasterMixin, RasterDataset):
```

Comment:
class looks good! Didn't know `ds["var"]` makes you loose attributes..

Reply:
Since the classes now rely on a set `x_dim` and `y_dim` rather than functions just specifying those explicitly, it is now a problem, but don't think it was previously (since we didn't set those I think, at least in code I've seen).